ide-eclipse.html

```diff
@@ -1,25 +1,25 @@
-<h2>JavaFX <span class="JFX_MAJOR">11</span> and Eclipse</h2>
+<h2>JavaFX and Eclipse</h2>
 
-<p>This section explains how to use Java <span class="JDK_MAJOR">11</span> and JavaFX <span class="JFX_MAJOR">11</span> from Eclipse.
-    Version: 2019-03 (4.11.0) was used for the following screenshots.
+<p>
+    This section explains how to create a JavaFX application in Eclipse.
+    JavaFX 12.0.2 and Eclipse 2019-03 (4.11.0) was used for the IDE screenshots.
 </p>
 
 <p>
     To get Java 12 support, install the patch from the Eclipse MarketPlace: <a href="https://marketplace.eclipse.org/content/java-12-support-eclipse-2019-03-411" target="_blank">Java 12 Support for Eclipse 2019-03 (4.11)</a>.
 </p>
 
 <p>
-    Download an appropriate <a href="https://jdk.java.net/12/">JDK <span class="JDK_MAJOR">11</span></a> for your operating system.
-    Make sure <kbd>JAVA_HOME</kbd> is properly set to the Java <span class="JDK_MAJOR">11</span> installation directory. 
+    Download an appropriate JDK for your operating system.
+    Refer to <a href="#install-java">Install Java</a> section for more information.
```

Comment:
Big change... imo JAVA_HOME, PATH, JDK 11+, are required terms, even if we repeat them elsewhere...

Reply:
How about "Download an appropriate JDK for your operating system and set `JAVA_HOME` to this JDK directory" ?

Reply:
ok